src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -12420,6 +12423,22 @@ LambdaExpressionSyntax parseLambdaExpressionWorker()
                     this.IsInAsync = true;
                 }
```

Comment:
is github just rendering this wrong.  i see:

![image](https://user-images.githubusercontent.com/4564579/121416265-750db800-c91d-11eb-8d8d-6ee98698e256.png)

i legit don't know how this is even legal c# code.  it looks like we parse the attribute... say we're in async... and then the method body ends?!  i'm assuming github is just broken here maybe?

Reply:
`ParseLambdaExpression()` contains several statements including a call to the `parseLambdaExpressionWorker()` local function where the actual changes are.